### PR TITLE
Unskip and fix MFTF tests

### PR DESCRIPTION
**Pull request by smoskaluk**
Verify skipped MFTF tests and unskip them

AdminAddMultipleSourcesToProductTest
AdminCreateSimpleProductReassignSource
AdminOutOfStockThresholdAppliedToSimpleInAdvancedInventoryOnProductPageTest
AdminOutOfStockThresholdOnConfigurationPageTest
AdminSelectDeselectManipulationsInManageSourceGridTest
GuestCustomerCreateOrderWithSimpleProductOnCustomStockTest

### Fixed Issues (if relevant)

1. magento/inventory#2565: Unskip skipped MFTF tests

### Manual testing scenarios (*)

none

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: app/code/Magento/InventoryAdminUi/Test/Mftf/Test/AdminOutOfStockThresholdAppliedToSimpleInAdvancedInventoryOnProductPageTest.xml [hunk 1]
```diff
@@ -17,114 +17,69 @@
...
-            <!--assign source and sales channel to stock-->
-            <amOnPage url="{{AdminManageStockPage.url}}" stepKey="amOnTheStockGridPage"/>
-            <waitForPageLoad time="30" stepKey="waitForStockGridPageLoad"/>
-            <actionGroup ref="searchAdminDataGridByKeyword" stepKey="searchCustomStockByKeyword">
-                <argument name="keyword" value="$$createCustomStock.stock[name]$$"/>
+            <createData entity="SimpleProduct" stepKey="simpleProduct">
+                <requiredEntity createDataKey="category"/>
+            </createData>
+            <amOnPage url="{{AdminProductEditPage.url($$simpleProduct.id$$)}}" stepKey="openProductEditPageToAssignSource"/>
+            <actionGroup ref="AssignSourceToProductActionGroup" stepKey="assignUSSource1ToCreatedProduct">
```
Comment (nmalevanec) [inline app/code/Magento/InventoryAdminUi/Test/Mftf/Test/AdminOutOfStockThresholdAppliedToSimpleInAdvancedInventoryOnProductPageTest.xml]: "assignUSSource1ToCreatedProduct" seem like copy-paste and not related to current test.

File: app/code/Magento/InventoryAdminUi/Test/Mftf/Test/AdminOutOfStockThresholdAppliedToSimpleInAdvancedInventoryOnProductPageTest.xml [hunk 2]
```diff
@@ -17,114 +17,69 @@
...
             </actionGroup>
-            <click selector="{{AdminGridRow.checkboxByValue($$createCustomSource.source[name]$$)}}" stepKey="clickOnTheCheckboxForSource"/>
-            <click selector="{{AdminManageSourcesGridControls.done}}" stepKey="clickOnDone"/>
-            <click selector="{{AdminGridMainControls.saveAndContinue}}" stepKey="saveCustomStock"/>
-
-            <amOnPage url="{{AdminProductIndexPage.url}}" stepKey="navigateToProductGrid"/>
+            <actionGroup ref="AdminChangeOutOfStockThresholdOnProductEditPageActionGroup" stepKey="setOutOfStockThreshold">
+                <argument name="threshold" value="100"/>
+            </actionGroup>
+            <click selector="{{AdminProductFormActionSection.saveButton}}" stepKey="saveEditedSimpleProduct"/>
```
Comment (nmalevanec) [inline app/code/Magento/InventoryAdminUi/Test/Mftf/Test/AdminOutOfStockThresholdAppliedToSimpleInAdvancedInventoryOnProductPageTest.xml]: There is action group for this action. Please use action group

File: app/code/Magento/InventoryAdminUi/Test/Mftf/Test/AdminOutOfStockThresholdAppliedToSimpleInAdvancedInventoryOnProductPageTest.xml [hunk 3]
```diff
@@ -17,114 +17,69 @@
...
+            </actionGroup>
+            <actionGroup ref="DisableSourceActionGroup" stepKey="disableCreatedCustomSource1">
+                <argument name="sourceCode" value="$$customSource.source[source_code]$$"/>
+            </actionGroup>
+            <deleteData createDataKey="category" stepKey="deleteCategory"/>
+            <deleteData createDataKey="CustomStockMainWebsite" stepKey="deleteStock"/>
+            <deleteData createDataKey="simpleProduct" stepKey="deleteProduct"/>
+            <magentoCLI command="indexer:reindex" stepKey="runIndexerReindex" />
+            <magentoCLI command="cache:flush" stepKey="cleanCache"/>
+            <waitForPageLoad time="30" stepKey="waitForIndexAndCache"/>
```
Comment (nmalevanec) [inline app/code/Magento/InventoryAdminUi/Test/Mftf/Test/AdminOutOfStockThresholdAppliedToSimpleInAdvancedInventoryOnProductPageTest.xml]: Move deleteData and magentoCli before AssignWebsiteToStockActionGroup and  DisableSourceActionGroup as mentioned here https://devdocs.magento.com/mftf/docs/tips-tricks.html
Beware of stock deletion. Stock cannot be deleted with website assigned, so stock should be deleted after AssignWebsiteToStockActionGroup

File: app/code/Magento/InventoryAdminUi/Test/Mftf/Test/AdminOutOfStockThresholdOnConfigurationPageTest.xml [hunk 4]
```diff
@@ -10,86 +10,74 @@
        xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
     <test name="AdminOutOfStockThresholdOnConfigurationPageTest">
         <annotations>
-            <stories value="Configuration Settings"/>
+            <stories value="Out-of-Stock Threshold on the Configuration page functions correctly"/>
```
Comment (nmalevanec) [inline app/code/Magento/InventoryAdminUi/Test/Mftf/Test/AdminOutOfStockThresholdOnConfigurationPageTest.xml]: Use for stories more generic lines: like Simple Product on Custom Stock, or Simple Product on Default Stock

File: app/code/Magento/InventoryAdminUi/Test/Mftf/Test/AdminOutOfStockThresholdOnConfigurationPageTest.xml [hunk 5]
```diff
@@ -10,86 +10,74 @@
...
+            </actionGroup>
+            <deleteData createDataKey="category" stepKey="deleteCategory"/>
+            <deleteData createDataKey="CustomStockMainWebsite" stepKey="deleteStock"/>
+            <deleteData createDataKey="simpleProduct" stepKey="deleteProduct"/>
+            <!-- Revert out of stock threshold value in config. -->
+            <magentoCLI command="config:set cataloginventory/item_options/min_qty 0" stepKey="setMinQtyToDefaultValue"/>
+            <magentoCLI command="indexer:reindex" stepKey="runIndexerReindex" />
+            <magentoCLI command="cache:flush" stepKey="cleanCache"/>
+            <waitForPageLoad time="30" stepKey="waitForIndexAndCache"/>
+            <actionGroup ref="logout" stepKey="logoutOfAdmin"/>
```
Comment (nmalevanec) [inline app/code/Magento/InventoryAdminUi/Test/Mftf/Test/AdminOutOfStockThresholdOnConfigurationPageTest.xml:60]: Move delete action and cli commands in first lines on after, as mentioned in previous comment

File: app/code/Magento/InventoryAdminUi/Test/Mftf/Test/GuestCustomerCreateOrderWithSimpleProductOnCustomStockTest.xml [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="GuestCustomerCreateOrderWithSimpleProductOnCustomStockTest">
+        <annotations>
+            <stories value="Guest Customer created Order with Simple product"/>
```
Comment (nmalevanec) [inline app/code/Magento/InventoryAdminUi/Test/Mftf/Test/GuestCustomerCreateOrderWithSimpleProductOnCustomStockTest.xml]: stories should be more generic line

File: app/code/Magento/InventoryAdminUi/Test/Mftf/Test/GuestCustomerCreateOrderWithSimpleProductOnCustomStockTest.xml [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+            <createData entity="BasicMsiStockWithMainWebsite1" stepKey="CustomStockMainWebsite"/>
+            <createData entity="SourceStockLinked1" stepKey="stockSourceLink">
+                <requiredEntity createDataKey="CustomStockMainWebsite"/>
+                <requiredEntity createDataKey="customSource"/>
+            </createData>
+            <createData entity="SimpleProduct" stepKey="simpleProduct">
+                <requiredEntity createDataKey="category"/>
+            </createData>
+            <amOnPage url="{{AdminProductEditPage.url($$simpleProduct.id$$)}}" stepKey="openProductEditPageToAssignSource"/>
+            <actionGroup ref="AssignSourceToProductActionGroup" stepKey="assignUSSource1ToCreatedProduct">
```
Comment (nmalevanec) [inline app/code/Magento/InventoryAdminUi/Test/Mftf/Test/GuestCustomerCreateOrderWithSimpleProductOnCustomStockTest.xml]: assignUSSource1ToCreatedProduct looks like copy-paste and not suited for this test

File: app/code/Magento/InventoryAdminUi/Test/Mftf/Test/GuestCustomerCreateOrderWithSimpleProductOnCustomStockTest.xml [hunk 8]
```diff
@@ -0,0 +1,104 @@
...
+            <waitForPageLoad time="20" stepKey="waitFroDefaultStockEditPageLoad"/>
+            <selectOption selector="{{AdminEditStockSalesChannelsSection.websites}}" userInput="Main Website" stepKey="selectDefaultWebsiteAsSalesChannelForDefaultStock"/>
+            <click selector="{{AdminGridMainControls.saveAndContinue}}" stepKey="saveDefaultStock"/>
+            <actionGroup ref="DisableSourceActionGroup" stepKey="disableCreatedCustomSource1">
+                <argument name="sourceCode" value="$$customSource.source[source_code]$$"/>
+            </actionGroup>
+            <actionGroup ref="logout" stepKey="logoutOfAdmin1"/>
+            <deleteData createDataKey="category" stepKey="deleteCategory"/>
+            <deleteData createDataKey="CustomStockMainWebsite" stepKey="deleteStock"/>
+            <deleteData createDataKey="simpleProduct" stepKey="deleteProduct"/>
```
Comment (nmalevanec) [inline app/code/Magento/InventoryAdminUi/Test/Mftf/Test/GuestCustomerCreateOrderWithSimpleProductOnCustomStockTest.xml]: See above comments about delete data

Comment (smoskaluk) [reply app/code/Magento/InventoryAdminUi/Test/Mftf/Test/GuestCustomerCreateOrderWithSimpleProductOnCustomStockTest.xml]: Done

**Review by nmalevanec — APPROVED** [review]

**Pull request merged**